runescape-client/src/main/java/Friend.java

```diff
@@ -61,15 +67,14 @@ public int compareTo(Object var1) {
 		signature = "(Lhp;IIB)Z",
 		garbageValue = "-3"
 	)
-	@Export("doesSpriteExist")
-	public static boolean doesSpriteExist(AbstractArchive var0, int var1, int var2) {
+	@Export("SpriteBuffer_loadSprite")
+	public static boolean SpriteBuffer_loadSprite(AbstractArchive var0, int var1, int var2) {
```

Comment:
`SpriteBuffer_cacheSprite` maybe? Want to make it obvious this doesn't return a `Sprite` instance based off its name, as that's why it got broke in 180 -> 181

Reply:
My vote is for bufferSprite. If I didn't have a problem with long method names I'd go for tryToLoadSpriteIntoBuffer but that's way too long imo. Maybe loadFile is even better actually, as it is loading pixels from a file

Reply:
+1 for `bufferSprite`